nixos/modules/services/misc/gitea.nix

```diff
@@ -446,8 +446,10 @@ in
         Type = "simple";
         User = cfg.user;
         Group = "gitea";
+        RuntimeDirectory = "gitea";
+        RuntimeDirectoryMode = "0755";
         WorkingDirectory = cfg.stateDir;
-        ExecStart = "${gitea}/bin/gitea web";
+        ExecStart = "${gitea}/bin/gitea web --pid /run/gitea/gitea.pid";
```

Comment:
@kolaente out of curiosity what impact will this have? I'm assuming it would have been better to have this all along?

Reply:
By default, Gitea puts its pid in `/var/run/gitea.pid`, not sure about other effects though

Reply:
While there shouldn't really be much of a reason to write PID files at all, gitea should really not use `/var/run`, but `/run`:https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s13.html

I opened https://github.com/go-gitea/gitea/pull/12391, and propose we don't set this here at all.

Reply:
By default, systemd creates a folder in / run. Need to move pifd file to /run/gitea/gitea.pid ?

Reply:
Should we really block on upstream here? Writing to `/run/gitea` does not look like it would break anything.